src/field_theory/tower.lean

```diff
@@ -81,6 +81,40 @@ let ⟨c, hc⟩ := exists_is_basis_finset K A in
 by rw [findim_eq_card_basis hb, findim_eq_card_basis hc,
     findim_eq_card_basis (hb.smul hc), fintype.card_prod]
 
+/-- Given a tower `F : K : A`, any submodule of `A` over `K` is also a submodule of `A` over `F`. -/
+def submodule_tower_submodule (p : submodule K A) : submodule F A :=
+{ carrier := p.carrier,
+  zero_mem' := p.zero_mem',
+  add_mem' := p.add_mem',
+  smul_mem' := begin
+    intros c x hx,
+    rw [← one_smul K x, ← smul_assoc],
+    exact p.smul_mem' (c • 1) hx,
+  end, }
+
+/-- Given a tower `F : K : A`, if `A` has finite dimension over `F` then it is also has
+  finite dimension over `K` -/
+lemma findim_of_tower_findim [finite_dimensional F A] : finite_dimensional K A :=
```

Comment:
Isn't this just `finite_dimensional.right`?

Reply:
You're right, as with `submodule.restrict_scalars` I just wasn't aware of `finite_dimensional.right`.